facility-manager.html

```diff
@@ -1149,11 +1139,9 @@
     Given the number of server operating systems, clients and components
     involved in OMERO, the matrix of versions and compatibility is large. If
     you are in any doubt about the suitability for upgrade of your particular
-    installation, draw on the wide experience of the OMERO community via the
-    <a href="http://www.openmicroscopy.org/community/" target="_blank"
-    class="external-url">OME Forums</a> or <a
-    href="http://www.openmicroscopy.org/site/community/mailing-lists"
-    target="_blank" class="external-url">OME Mailing Lists</a>, giving as much
+    installation, draw on the wide experience of the OMERO community visit the
+    <a href="https://www.openmicroscopy.org/support/" target="_blank"
+    class="external-url">Support</a> page, giving as much
```

Comment:
"giving as much detail as you can" doesn't fit well following "visit the support page", we need to add back in reference to the mailing lists/forums:

``...OMERO community via the <a href="https://www.openmicroscopy.org/support/" target="_blank"
 +    class="external-url">forums or mailing lists</a> page, giving as much...``